torch_geometric/data/batch.py

```diff
@@ -128,16 +192,12 @@ def get_example(self, idx: int) -> BaseData:
 
         return data
 
-    def index_select(self, idx: IndexType) -> List[BaseData]:
-        r"""Creates a subset of :class:`~torch_geometric.data.Data` or
-        :class:`~torch_geometric.data.HeteroData` objects from specified
-        indices :obj:`idx`.
-        Indices :obj:`idx` can be a slicing object, *e.g.*, :obj:`[2:5]`, a
-        list, a tuple, or a :obj:`torch.Tensor` or :obj:`np.ndarray` of type
-        long or bool.
-        The :class:`~torch_geometric.data.Batch` object must have been created
-        via :meth:`from_data_list` in order to be able to reconstruct the
-        initial objects.
```

Comment:
This still seems to be the case here - or am I missing something?

Reply:
My bad, that must have been a mistake.